pkg/js/validator/validate-rules.ts

```diff
@@ -0,0 +1,46 @@
+export const Rules = {
+  type: "[^:#@\\s]{1,254}",
+  relation: "[^:#@\\s]{1,50}",
+  condition: "[^\\s]{2,256}",
+  object: "[^\\s]{2,256}",
+  id: "[^#:\\s]+",
+};
```

Comment:
If we have any validation on the server for the condition param name, let's add it here